src/App.tsx

```diff
@@ -0,0 +1,59 @@
+import './main.scss';
+
+import { useSelector } from 'react-redux';
+import { AppRouter } from './apps/app-router';
+import { ZUIProvider } from '@zero-tech/zui/ZUIProvider';
+import { RootState } from './store';
+import classNames from 'classnames';
+import { getMainBackgroundClass, getMainBackgroundVideoSrc } from './utils';
+import { AppBar } from './components/app-bar';
+import { DialogManager } from './components/dialog-manager/container';
+import { ThemeEngine } from './components/theme-engine';
+
+export const App = () => {
```

Comment:
Function components as these components are very simple. But, happy to change to class components if need be.